### PR TITLE
fix: sr18 search results bug

### PR DESCRIPTION
**Pull request by Heleen-cr**
fixes: #377

### CODE REVIEW

**Comment by Heleen-cr** [comment]
SR environment: http://pr-185-6ylsgva-mw3yu47main2e.eu.platform.sh/

File: Gruntfile.js [hunk 1]
```diff
@@ -410,6 +428,7 @@ module.exports = function (grunt) {
   grunt.registerTask('build:cr17', [
     'sass:cr17',
     'modernizr',
+    'imagemin:cr17',
```
Comment (AndyEPhipps) [inline Gruntfile.js]: Mmmm do we need this update for this SR work?

File: sass/base/_components.scss [hunk 2]
```diff
@@ -48,6 +48,7 @@
 @import "components/promo-header/promo-header";
 @import "components/quote-block/content-wall-quote";
 @import "components/related-articles/related-articles";
+@import "components/schools-look-up/schools-look-up";
```
Comment (AndyEPhipps) [inline sass/base/_components.scss]: Again, do we need this as part of this PR?

File: sass/themes/sr/2018/components/search-results/_search-results.scss [hunk 3]
```diff
@@ -0,0 +1,41 @@
+.path-search {
+  background-color: $colour-white;
+  .search-block {
+    position: relative;
+    padding-top: 0;
+  }
+  .search-block__form {
+    margin: 0 auto;
+    form {
+      @include breakpoint($screen-md){
```
Comment (AndyEPhipps) [inline sass/themes/sr/2018/components/search-results/_search-results.scss]: lol space here before the brace plz

File: sass/themes/sr/2018/components/search/_search-block.scss [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+  // offset from navigation
+  position: fixed;
+  padding-top: $sm-nav-height;
+  @include breakpoint($screen-md) {
+    padding-top: $md-nav-height;
+  }
+  @include breakpoint($desktop-nav-breakpoint) {
+    padding-top: $lg-nav-height;
+  }
+  @include gradient-background($colour-purple, $colour-ruby);
```
Comment (AndyEPhipps) [inline sass/themes/sr/2018/components/search/_search-block.scss]: Let's bung this up top with the other rules

File: sass/themes/sr/2018/components/search/_search-block.scss [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+    margin-bottom: 5px;
+  }
+  input[type=text] {
+    height: 50px;
+    padding: 0;
+    font-family: $font-bold--sr18;
+    @include font-size($p-large-font-size);
+    color: $colour-white;
+    background-color: transparent;
+    @include breakpoint($screen-md){
```
Comment (AndyEPhipps) [inline sass/themes/sr/2018/components/search/_search-block.scss]: spaaaaace

File: Gruntfile.js (hunk 1 above)
Comment (Heleen-cr) [reply Gruntfile.js]: probably not, will remove for now

File: sass/base/_components.scss (hunk 2 above)
Comment (Heleen-cr) [reply sass/base/_components.scss]: that was a merge from master that had a little hiccup and I had to commit myself

**Pull request merged**